squad-server/plugins/discord-admin-request.js

```diff
@@ -123,9 +134,37 @@ export default class DiscordAdminRequest extends DiscordBasePlugin {
 
     await this.sendDiscordMessage(message);
 
-    await this.server.rcon.warn(
-      info.player.steamID,
-      `An admin has been notified, please wait for us to get back to you.`
-    );
+    const admins = await this.server.getAdminsWithPermission('canseeadminchat');
+    let amountAdmins = 0;
+    if (this.options.warnInGameAdmins) {
+      for (const player of this.server.players) {
+        if (!admins.includes(player.steamID)) continue;
+        amountAdmins++;
+        await this.server.rcon.warn(player.steamID, `[${info.player.name}] - ${info.message}`);
+      }
+
+      if (amountAdmins === 0 && this.options.showInGameAdmins)
+        await this.server.rcon.warn(
+          info.player.steamID,
+          `There are no in-game admins, however an admin has been notified via discord, please wait for us to get back to you.`
+        );
+      else if (this.options.showInGameAdmins)
+        await this.server.rcon.warn(
+          info.player.steamID,
+          `There ${amountAdmins > 1 ? 'are' : 'is'} ${amountAdmins} in-game admin${
+            amountAdmins > 1 ? 's' : ''
+          }. Please wait for us to get back to you.`
+        );
+      else
+        await this.server.rcon.warn(
+          info.player.steamID,
+          `An admin has been notified, please wait for us to get back to you.`
+        );
```

Comment:
Thinking about it, should this be dependent on the `warnInGameAdmins` option? Id's suggest making it so you can have this enabled without `warnInGameAdmins` being enabled.